documentation/api/v1/async_callback_api_doc.rb

```diff
@@ -0,0 +1,151 @@
+require 'swagger/blocks'
+
+module AsyncCallbackApiDoc
+  include Swagger::Blocks
+
+  UUID_DESC = 'The unique ID of the payload calling back'
+  UUID_EXAMPLE = '6dde5ce0e94c9f43'
+  TIMESTAMP_DESC = 'The Unix format timestamp when the asynchronous payload called back (in epoch)'
+  TIMESTAMP_EXAMPLE = '1536777407'
+  LISTENER_URI_DESC = 'The URL of the listener which received the callback'
+  LISTENER_URI_EXAMPLE = 'tcp://192.168.1.7:4444'
+  HOST_DESC = 'The IP address from which the callback was received'
+  HOST_EXAMPLE = '203.0.113.5'
+  PORT_DESC = 'The port from which the callback was received'
+  PORT_EXAMPLE = '443'
+  WORKSPACE_ID_DESC = 'The ID of the workspace this payload belongs to'
+  WORKSPACE_ID_EXAMPLE = 'default'
+
+# Swagger documentation for payloads model
+  swagger_schema :AsyncCallback do
+    key :required, [:ntype]
+    property :workspace, type: :string, required: true, description: RootApiDoc::WORKSPACE_POST_EXAMPLE
+    property :id, type: :integer, format: :int32, description: RootApiDoc::ID_DESC
+    property :uuid, type: :string, required: true, description: UUID_DESC, example: UUID_EXAMPLE
+    property :timestamp, type: :integer, required: true, description: TIMESTAMP_DESC, example: TIMESTAMP_EXAMPLE
+    property :listener_uri, type: :string, description: LISTENER_URI_DESC, example: LISTENER_URI_EXAMPLE
+    property :target_host, type: :string, description: HOST_DESC, example: HOST_EXAMPLE
+    property :target_port, type: :string, description: PORT_DESC, example: PORT_EXAMPLE
+    property :created_at, type: :string, format: :date_time, description: RootApiDoc::CREATED_AT_DESC
+    property :updated_at, type: :string, format: :date_time, description: RootApiDoc::UPDATED_AT_DESC
+  end
+
+  swagger_path '/api/v1/async-callbacks' do
```

Comment:
The endpoint `/api/v1/async-callbacks` is very generic sounding, please consider adding the term `payload` into the URL to provide more context. (See other uses throughout PR)